lib/src/models/message.dart

```diff
@@ -91,25 +91,61 @@ class Message {
   }
 
   factory Message.fromJson(Map<String, dynamic> json) => Message(
-      id: json["id"],
-      message: json["message"],
-      createdAt: json["createdAt"],
-      sendBy: json["sendBy"],
-      replyMessage: ReplyMessage.fromJson(json["reply_message"]),
-      reaction: Reaction.fromJson(json["reaction"]),
-      messageType: json["message_type"],
-      voiceMessageDuration: json["voice_message_duration"],
-      status: json['status']);
+        id: json['id']?.toString() ?? '',
+        message: json['message']?.toString() ?? '',
+        createdAt:
+            DateTime.tryParse(json['createdAt'].toString()) ?? DateTime.now(),
+        sendBy: json['sendBy']?.toString() ?? '',
+        replyMessage: json['reply_message'] is Map<String, dynamic>
+            ? ReplyMessage.fromJson(json['reply_message'])
+            : const ReplyMessage(),
+        reaction: json['reaction'] is Map<String, dynamic>
```

Comment:
You can write this condition inside the constructor parameter itself rather than writing `Reaction.fromJson` twice.